docs/01_setup.md

```diff
@@ -58,9 +58,86 @@ To create your own round you need to do a few things:
 
 If you are running on a different network than Optimism you need to update the contract addresses for EAS. These addresses are used whenever an attestation is created.
 
-You can also configure your own schemas here if you wish to.
+You can also configure your own schemas here if you wish to, or deploy the EAS contracts in a network that doesn't have it.
 
-## 3. Deploy
+## 3. Deploy MACI
+
+TODO: improve this,  test maci-cli setup
+
+As a coordinator you need to deploy MACI instance and poll.
+
+### Install Maci
+
+You can read about the [MACI requirements here](https://maci.pse.dev/docs/installation). To install MACI run the following commands:
+
+```bash
+git clone https://github.com/privacy-scaling-explorations/maci.git && \
+cd maci && \
+pnpm i && \
+pnpm run build
+```
+
+### Download .zkey files
+
+Note the locations of the .zkey files as the CLI requires them as command-line flags. Download ceremony artifacts:
+
+```bash
+pnpm download:ceremony-zkeys
+```
+
+### Set .env Files
+
+Head to the `cli` folder and copy the `.env.template` file. Make sure to include a private key and RPC url.
+
+```
+ETH_SK="your_ethereum_secret_key"
+ETH_PROVIDER="the_eth_provider_url"
+```
+
+### Generate MACI Keys
+
+Generate a new key pair and save it in a secure place.
+
+```bash
+node build/ts/index.js genMaciKeyPair
+```
+
+### Deploy MACI Contracts
+
+The following scripts deploy a MACI instance and set the keys, make sure to save the address of the MACI contract.
+
+> [!IMPORTANT]
+> Make sure that you use the NonQv keys
```

Comment:
question: they could only use nonQV version, or actually they can choose to deploy QV or nonQV one?

Reply:
Maybe we should try to run a QV round to see what happens